### PR TITLE
Updated pull request (1207) platform.h

### PR DESCRIPTION
**Pull request by lalauria2000**
on behalf of @wallarug. Part 3 of 3.
changes for the platform.h in common packages
See: https://github.com/microsoft/pxt-common-packages/pull/1207
For rest of files: https://github.com/microsoft/pxt-maker/pull/329

### CODE REVIEW

File: libs/core---samd/platform.h [hunk 1]
```diff
@@ -135,4 +135,73 @@ typedef int PinName;
 #define PB30 62
 #define PB31 63
 
+
+// 128 pin packages
+#ifdef SAMD51
```
Comment (pelikhan) [inline libs/core---samd/platform.h:140]: @mmoskal i assume we always want those so probably remove this #ifdef

Comment (mmoskal) [reply libs/core---samd/platform.h]: yes

**Review by mmoskal — APPROVED** [review]

**Comment by mmoskal** [comment]
Thank you!

**Comment by lalauria2000** [comment]
you're welcome! I was wondering, does dal.d.ts need to be updated with PC and PD ports too?

**Comment by mmoskal** [comment]
well, yes, but that happens in particular editors (pxt-maker, pxt-arcade etc)

**Comment by lalauria2000** [comment]
okay got it. Whenever you have time please check this with @pelikhan
https://github.com/microsoft/pxt-maker/pull/329  which is the follow up to this issue 
https://github.com/microsoft/pxt-maker/issues/328
Thanks!

**Pull request merged**